Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6181,6 +6182,38 @@ def com_google_fonts_check_colorfont_tables(ttFont):
     else:
         yield PASS, "Looks good!"
 
+@check(
+    id = "com.google.fonts/check/empty_glyph_on_gid1_for_colrv0",
+    rationale = """
+        A rendering bug in Windows 10 paints whichever glyph is on GID 1
+        on top of some glyphs, colored or not. This only occurs for COLR version 0 fonts.
+
+        Having a glyph with no contours on GID 1 is a practical workaround for that.
+
+        See https://github.com/googlefonts/gftools/issues/609
+    """
+)
+def com_google_fonts_check_empty_glyph_on_gid1_for_colrv0(ttFont):
+    """Put an empty glyph on GID 1 right after the .notdef glyph for COLRv0 fonts."""
+    SUGGESTED_FIX = ("To fix this, please reorder the glyph order so that"
+                    " a glyph with no contours is on GID 1 right after the .notdef glyph."
+                    " This could be the space glyph.")
+    from fontTools.pens.areaPen import AreaPen
+    glyphSet = ttFont.getGlyphSet()
+    pen = AreaPen(glyphSet)
+    gid1 = glyphSet[glyphSet.keys()[1]]
```

Comment:
I don't think the glyphset returned from getGlyphSet is guaranteed to be sorted by glyph order. It's preferable that you first getGlyphOrder() and then take the second glyph from it and then you get it from the glyph set dictionary.